db/view/view.cc

```diff
@@ -2285,14 +2286,16 @@ class view_builder::consumer {
     // beyond our limit on mutation size (by default 32 MB).
     size_t _fragments_memory_usage = 0;
 public:
-    consumer(view_builder& builder, build_step& step, gc_clock::time_point now)
+    consumer(view_builder& builder, shared_ptr<view_update_generator> gen, build_step& step, gc_clock::time_point now)
             : _builder(builder)
+            , _gen(std::move(gen))
             , _step(step)
             , _built_views{step}
             , _now(now) {
         if (!step.current_key.key().is_empty(*_step.reader.schema())) {
             load_views_to_build();
         }
+        (void)_gen;
```

Comment:
What's the purpose of this line?

Reply:
Argh, it's compiler-silencing trick. When `_gen` is first introduced it's unused and compilter warns about it. Few patches later it becomes used and should be removed. I'll post the update